testing/simapp/app.go

```diff
@@ -577,7 +576,6 @@ func NewSimApp(
 
 	// initialize stores
 	app.MountKVStores(keys)
-	app.MountTransientStores(tkeys)
```

Comment:
Is it no need?

Reply:
Finschia's baseapp doesn't have `MountTransientStores`.